doc/manuals/admin/management_api.md

```diff
@@ -7,14 +7,15 @@ API for management that allows to change the trace and verbosity levels
 In order to manage the trace level:
 
 ```
-curl --request DELETE <host>:<port>/log/trace
-curl --request DELETE <host>:<port>/log/trace/t1
-curl --request DELETE <host>:<port>/log/trace/t1-t2
-curl --request DELETE <host>:<port>/log/trace/t1-t2,t3-t4
-curl --request GET <host>:<port>/log/trace
-curl --request PUT <host>:<port>/log/trace/t1
-curl --request PUT <host>:<port>/log/trace/t1-t2
-curl --request PUT <host>:<port>/log/trace/t1-t2,t3-t4
+curl --request DELETE <host>:<port>/log/trace
+curl --request DELETE <host>:<port>/log/trace/t1
+curl --request DELETE <host>:<port>/log/trace/t1-t2
+curl --request DELETE <host>:<port>/log/trace/t1-t2,t3-t4
+curl --request GET <host>:<port>/log/trace
+curl --request PUT <host>:<port>/log/trace/t1
+curl --request PUT <host>:<port>/log/trace/t1-t2
+curl --request PUT <host>:<port>/log/trace/t1-t2,t3-t4
```

Comment:
Don't know why git shows the curl has change, but looking to them, are the same.

NTC